### PR TITLE
Adjust docs for chaos worker

### PR DESCRIPTION
**Pull request by Zelldon**
Add more details to readme's

closes https://github.com/zeebe-io/zeebe-cluster-testbench/issues/87

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -260,6 +260,7 @@ This workflow creates a Zeebe cluster in Camnuda cloud and waits until the clust
 | Destroy Zeebe Cluster in Camunda CLoud | `destroy-zeebe-cluster-in-camunda-cloud` / `destroy-zeebe-cluster-in-camunda-cloud-job` | `clusterId` |
 | Create Generation in Camunda Cloud | `create-generation-in-camunda-cloud` / `create-generation-in-camunda-cloud-job` | `zeebeImage`, `generationTemplate`, `channel` | `generation`, `generationUUID` | |
 | Delete Generation in Camunda Cloud | `delete-generation-in-camunda-cloud` / `delete-generation-in-camunda-cloud-job` | `generationUUID` | | |
+| Chaos Experiment | `chaos-experiments` / `chaos-experiments` | `authenticationDetails`, `clusterPlan` | `testResult`, `testReport` |
```
Comment (pihme) [inline README.md:263]: I think we should rename this to _Chaos Experiments_ (here and in the BPMN files)

Not sure if you want to do this now or sometime later

File: core/chaos-workers/README.md [hunk 2]
```diff
@@ -13,5 +13,17 @@ On building the docker image a service account token needs to be given, which is
 When running the image a zeebe job worker is started, as environment variables the auth details to the testbench have to been set.
 
 The job worker runs a job handler script. This handler extracts from the given variables all necessary informations like auth details of the targeting cluster but also which cluster plan is it.
-Regarding of the cluster plan different chaos experiments are executed. The job is completed with a `PASSED` or `FAILED` testResult variable.
+Depending of the cluster plan different chaos experiments are executed. The job is completed with a testReport.
```
Comment (pihme) [inline core/chaos-workers/README.md:16]:
```suggestion
Depending on the cluster plan different chaos experiments are executed. The job is completed with a `testReport`.
```

File: core/chaos-workers/README.md [hunk 3]
```diff
@@ -13,5 +13,17 @@ On building the docker image a service account token needs to be given, which is
...
+
+The report contains a result which is either `PASSED`, `FAILED` or `SKIPPED`.
+
+ * `PASSED` means that all chaos experiment's have been completed successfully. In the metadata of the test report the experiments are listed, which have been executed.
+ * `FAILED` means that **one** experiment has failed. In the metadata of the test report the experiments are listed, which have been executed. Which experiment has failed can be found in the failure messages.
+ * `SKIPPED` means that no experiments has been found and that the execution has been skipped
```
Comment (pihme) [inline core/chaos-workers/README.md:22]:
```suggestion
 * `SKIPPED` means that no experiments have been found and that the execution has been skipped
```

File: core/chaos-workers/README.md [hunk 4]
```diff
@@ -13,5 +13,17 @@ On building the docker image a service account token needs to be given, which is
...
+ 
+## Chaos Worker
```
Comment (pihme) [inline core/chaos-workers/README.md:24]: Maybe move this over to the technical documentation. But it can also stay here for a time being.

**Review by pihme — APPROVED** [review]
Some suggestions and comments. Feel free to implement or ignore them.

File: core/chaos-workers/README.md (hunk 4 above)
Comment (Zelldon) [reply core/chaos-workers/README.md:24]: Hm for me it feels more intuitive to have the doc/readme which describes it next to the code.

File: README.md (hunk 1 above)
Comment (Zelldon) [reply README.md:263]: You mean the name of the ServiceTask?

Comment (pihme) [reply README.md:263]: The name of the service task, 
the name of the call activity in chaos test workflow https://raw.githubusercontent.com/zeebe-io/zeebe-cluster-testbench/master/docs/assets/chaos-test.png

And maybe the name of the call activity in run all tests https://raw.githubusercontent.com/zeebe-io/zeebe-cluster-testbench/master/docs/assets/run-all-tests-in-camunda-cloud.png

Comment (Zelldon) [reply README.md:263]: I will rename it to `Run Chaos Experiments` ok?

**Comment by Zelldon** [comment]
Hey @pihme I updated the workflows and images. What I not updated was the name of the workflow. The pattern here is similar to the sequential test workflow. Any opinion on this? Should I also rename it?

**Comment by pihme** [comment]
I don't have a clear preference. I am fine either way.

**Pull request merged**